### DISCUSSION TITLE
I wonder how your http benchmark performed

### DISCUSSION THREAD

**Discussion by skyitachi** [uNetworking/uWebSockets#893] [Q&A] [answered]
did the http benchmark use the `benchmarks/load_test.c`? if not, can you publish the how the benchmark performs?

**Comment by ghost** [comment]
Wrk was used

**Comment by skyitachi** [comment]
I mean your test fixture，use load_test.c as test server?

**Comment by ghost** [comment, marked as answer]
Wrk cannot stress uWS fully, use uSockets http_load_test to squeeze a few extra tens of thousands of req/sec
